libevent-sys/Cargo.toml

```diff
@@ -16,7 +16,9 @@ build = "build.rs"
 
 [features]
 static = []
+bundled = [ "static", "cmake" ]
 
 [build-dependencies]
 bindgen = "^0.45"
+cmake = { version = "0.1", optional = true }
 pkg-config = "^0.3"
```

Comment:
Should `pkg-config` be optional and only enabled if using the `static` feature?

Reply:
Hmm, I think that would mean either `default-features = false` wouldn't be supported by `libevent-sys`, or introducing the need for handling features `pkg-config` and `bundled` as non-mutually-exclusive.